src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -78,6 +78,7 @@
     "django-model-utils": ("model_utils",),
     "django-ninja": ("ninja",),
     "django-ninja-extra": ("ninja_extra",),
+    "django-postgres-extra": ["psqlextra"],
```

Comment:
would be nice to use a tuple here to match the other entries.

Reply:
```suggestion
    "django-postgres-extra": ("psqlextra",),
```